common.blocks/desktop/popup/login-form/login-form.css

```diff
@@ -0,0 +1,9 @@
+.login-form{
+    display: none;
+    flex-direction: column;
+    width: 800px;
+    height: 350px;
+    position: absolute;
+    top: calc(100% / 2 - 175px);
+    left: calc(100% / 2 - 400px);
+}
```

Comment:
You have the same styles for .sign-up form. It's better to use common class to style them (btw, you have no reason to set absolute height for them)